packages/themes/ecl/src/ecl-ec/components/pagination.scss

```diff
@@ -13,6 +13,18 @@
 		text-decoration: none;
 	}
 
+	.kol-pagination {
+		&__button {
+			&--selected .button-inner {
```

Comment:
Bitte analog zu meinem Beispiel im Default-Theme die Styles in den button-inner Code-Block verschieben.